### DISCUSSION TITLE
2021-09-14-SecretDecks Meeting

### DISCUSSION THREAD

**Discussion by DanM3rcurius** [SecretDecks/Documentation#102] [Meetings]
# SD Meeting 14th Sept. 9pm  - 11:30 pm (UTC+2)
Heather, Kat, Mars, Sara, GL, Tyler, Dan

## Agenda
- Founders Drop Content & SmartContract sale?
- Website relaunch & Founders Drop rollout
- Twitter Giveaway
- Open Discussions

### Founders Drop Content
- TODO: final edits in wording, animation approval!
	- maybe add edge effect?!
	- change speed? make it **a bit slower @tamagit80**  , then 15 second length
	- alternate speeds? for future unlocked states yes maybe, after going through "portal" smart contract
	
- SmartContract from TAC? yey or ney? 
	- Yes, @tamagit is going to setup, ASAP next days

### Website relaunch (Founders Drop Rollout)
- TODO: go over wording and lore, do **final edits! @evolayak** 
- next steps: finalize content, create and style pages in backend, enter content
- Discuss welcome/summary claim: "Virtual  collectible Tarot cards with layered utility focused around knowledge of Self"

- TO DO : STAGING ENV! - in GitHub?! **yes please @tamagit80** 

### Twitter Giveaway
- Tyler helped Trybbles, platform to use: gleam.io or givelabs.com
- hype up drop by giving out NFT before? 
- mint additional NFTs for giveaways and contributions; take out one base element as giveaway & contribution element - airdrop or pull? create new elements? **poll! @DanM3rcurius to do** 

### Open Discussions
- Artists:  do we want commission work done in beginning with pre payment? -> **Start Discussion @DanM3rcurius**
- SPO - we dont need 1 million? no, we need lots of delegators only! We need a good story!
- Intro for Mars V. what we been up to, Fund6 proposals, voting, SourceCred, founders drop & website relaunch
- approaching artist on IG @ MarsV(MC)
- Mars and Dan created two new elemets!

### To Do's
- [x] @DanM3rcurius  write down website workflow
- [ ] @DanM3rcurius Start discussion about Artist commission works 
- [ ] @DanM3rcurius  poll if we want to create new elements or take out one of the tier 1 for giveaways and rewards 
- [ ] @tamagit80 Staging environment & repo (on github) for website 
- [ ] @tamagit80 make animation a bit slower
- [ ] @wlstylr setup giveaway
- [ ] @evolayak final edits for website content (in gitbook)

**Comment by mars** [comment]
I think y'all have the wrong @mars 🙈

**Reply by DanM3rcurius** [reply]
> I think y'all have the wrong @mars 🙈

Ha yes. I'm sorry - forgot to edit the handle in the minutes 😅
